client/src/Components/App/App.js

```diff
@@ -72,7 +72,8 @@ const App = () => {
           return <JournalEntry journal={journal}/>
         }}/>
         <Route exact path='/how-you-felt'><FeelingsLog logs={userLogs}/></Route>
-        <Route path='*'><NotFound /></Route>
+        <Route exact path='/404'><NotFound /></Route>
+        <Route path='*'><Redirect to='/404'/></Route>
```

Comment:
+1 to CRC! Love this! Would love to hear more about your understanding of how `<Redirect>` works at the next standup!